docs/api-reference/io-columns-regression.md

```diff
@@ -0,0 +1,6 @@
+### Input and Output Columns
```

Comment:
is the file name for this too generic? Maybe io-sdca-columns-regression.md? That way its obvious this is used for sdca